CHANGELOG.md

```diff
@@ -1,3 +1,106 @@
+# 5.0
+## Breaking Changes
+* **Removed** support for `PascalCaseCustomRenderers` in view schema definitions
+* **Added** support for `kebab-case-custom-renderers` in view schema definitions
+  this was done mainly to accommodate a terser DSL in view schemas for built-in
+  renderers such as `select` which was added in this version. The goal is to have
+  simple built-in renderers be targeted with easy names like `select`, `buckets`, etc
+  like they were in UI Schema. UI Schema 2 will follow suit as well.  True custom renderers
```

Comment:
Referencing UI Schema on an open source project will likely confuse people. This description should probably be revised a little bit.